app/core/core-datastore-public/src/main/kotlin/com/hedvig/android/core/datastore/SettingsDataStore.kt

```diff
@@ -8,24 +8,22 @@ import com.hedvig.android.theme.Theme
 import kotlinx.coroutines.flow.Flow
 import kotlinx.coroutines.flow.map
 
-interface SettingsDataStore {
-  suspend fun setTheme(theme: Theme)
-  fun observeTheme(): Flow<Theme>
-}
-
-internal class SettingsDataStoreImpl(
+class SettingsDataStore(
```

Comment:
Testing this by faking the datastore storage https://github.com/HedvigInsurance/android/blob/feaf403b4550d25d005ff5bacc33e52a684faaf8/app/feature/feature-profile/src/test/kotlin/com/hedvig/android/feature/profile/settings/SettingsPresenterTest.kt#L25-L30 instead of faking the UseCase itself was more convenient here.

Not always sure which way to go with these, but testing the real implementation with a fake in-memory DataStore is really convenient, and you're testing with the real implementation. In the tests I used it for here it was convenient enough that not having an interface to create a test double here made sense.